shared-module/storage/__init__.c

```diff
@@ -109,19 +109,20 @@ void common_hal_storage_umount_object(mp_obj_t vfs_obj) {
     mp_vfs_proxy_call(vfs, MP_QSTR_umount, 0, NULL);
 }
 
-void common_hal_storage_umount_path(const char* mount_path) {
-    // remove vfs from the mount table
-    mp_obj_t *vfs_obj = NULL;
+STATIC mp_obj_t storage_object_from_path(const char* mount_path) {
     for (mp_vfs_mount_t **vfsp = &MP_STATE_VM(vfs_mount_table); *vfsp != NULL; vfsp = &(*vfsp)->next) {
         if (strcmp(mount_path, (*vfsp)->str) == 0) {
-            vfs_obj = (*vfsp)->obj;
+            return (*vfsp)->obj;
             break;
```

Comment:
No need to have a break after a return.

Reply:
Cleaned it up in one of my rebases.  Thanks.